kolibri/core/auth/test/test_api.py

```diff
@@ -926,6 +927,100 @@ def test_sign_up_able_no_guest_access(self):
         self.assertTrue(models.FacilityUser.objects.all())
 
 
+class PublicSignUpTestCase(APITestCase):
+    @classmethod
+    def setUpTestData(cls):
+        cls.facility = FacilityFactory.create()
+        provision_device()
+
+    def post_to_sign_up(self, data):
+        return self.client.post(
+            reverse("kolibri:core:publicsignup-list"), data=data, format="json"
```

Comment:
Not sure what pattern we use elsewhere, but it might be wise to have tests of public endpoints actually be hardcoded to make requests to a fixed path, since URL reversal would mask any accidental undesired changes to the path.

(Alternatively, adding explicit tests that the URL reversal is equal to a particular path)